app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Comments < ApplicationController
+
+def index
+  @commennts = Comments.all
```

Comment:
Couple things...
* It should be `Comment`, not `Comments`. Very important you use singular notation when referencing the class/model name.
* You misspelled `@comments`. That could cause problems elsewhere if you're not consistent with naming.